index.html

```diff
@@ -686,6 +686,51 @@ <h4>Redirected and Proxied External Content</h4>
           </p>
         </section>
       </section>
+
+      <section id="state-tokens">
+        <h3>State Tokens</h3>
+        <blockquote class="informative">
+          Non-normative note: State tokens are similar to ETags, providing a mechanism for clients to track changes
+          in resource state without having to compare the entire entity body, and to perform conditional updates.
+          Like ETags, the principle behind state tokens is that implementations have the best information on
+          resource state, and can choose the most accurate and efficient validation mechanism. State tokens differ
```

Comment:
I find the use of "validation" jarring as it suggests to me something more than what a state token does. Perhaps "validation mechanism" -> "change detection mechanism" (although really we want to talk about detecting NO change)